spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -2999,11 +3058,37 @@ void dump_spdm_chunk_response(const void *buffer, size_t buffer_size)
                spdm_response->header.param2,
                spdm_response->chunk_seq_no,
                spdm_response->chunk_size);
+
         if (spdm_response->chunk_seq_no == 0) {
+            /*first chunk*/
             large_message_size = (void *)(spdm_response + 1);
             printf(", LargeMsgSize=0x%08x", *large_message_size);
+            responder_send_large_message = malloc(*large_message_size);
```

Comment:
1) please check if malloc success or not.

2) please check if it is not NULL before malloc, and free if it is NOT nul.